src/pages/en/guides/images.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Images
+description: Learn how to use images in Astro.
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+Astro provides several ways for you to use images on your site, whether they are stored locally inside your project, linked to remotely, or stored in a CMS or CDN!
+
+### In `.astro` files
+
+Astro uses standard HTML `<img>` or `<img />` elements to display images within your `.astro` files. All HTML image attributes are supported.
+
+```astro
+---
+// src/pages/index.astro
+import rocket from './images/rocket.svg';
+---
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro">
+<img src="/assets/stars.png" alt="A starry night sky."> <!-- public/assets/stars.png -->
+<img src="./images/astronaut.jpg" alt="An astronaut"> <!-- src/images/astronaut.jpg -->
+<img src={rocket} alt="A rocketship in space."/> <!-- src/images/rocket.svg -->
+```
+
+### In .`md` and `.mdx` files
+
+You can use standard Markdown `![]()` syntax in your `.md` and `.mdx` files. You can also use HTML `<img>`(in Markdown) or JSX `<img />` (in MDX) if preferred.
+
+
+```md
+// src/pages/post-1.md
+
+<!-- src/images/astronaut.jpg -->
+![An astronaut](./images/astronaut.png)
+<img src="./images/astronaut.jpg" alt="An astronaut">
+
+ <!-- public/assets/stars.png -->
+![A starry night sky.](/assets/stars.png)
+<img src="/assets/stars.png" alt="A starry night sky.">
+
+![Astro](https://astro.build/assets/logo.png)
+```
+
+
+```mdx
+// src/pages/post-2.mdx
+---
+image: './images/rocket.svg'
+---
+
+<!-- src/images/astronaut.jpg -->
+![An astronaut](./images/astronaut.png)
+<img src="./images/astronaut.jpg" alt="An astronaut" />
+
+
+<!-- public/assets/stars.png -->
+![A starry night sky.](/assets/stars.png)
+<img src="/assets/stars.png" alt="A starry night sky." />
+
+<img src={frontmatter.image} alt="A rocketship in space." />
+
+![Astro](https://astro.build/assets/logo.png)
+```
+
+### In UI Framework Components
+
+In a [UI framework component](/en/core-concepts/framework-components/), write your images as appropriate for its own syntax.
+
+## Where to keep images
+
+### `src/`
+Your images kept in `src/` can be imported and used by other components, and will be processed during Astro's build process. These images are referenced **by relative [file path](https://www.codecademy.com/resources/docs/html/file-paths)**. 
+
+
+```astro
+---
+// src/pages/index.astro
+
+// Access images in `src/images/`
+---
+<img src="../images/logo.png" />
+```
```

Comment:
Already discussed on Discord: worth double checking this works on build as well as dev